### PR TITLE
Add Summary Bar

### PR DESCRIPTION
**Pull request by rememberlenny**
## What

Add new designs for Summary Bar.

## Trello

https://trello.com/c/KS9DRViS/373-details-summary-bar

### CODE REVIEW

File: app/views/proposals/details/_summary.html.haml [hunk 1]
```diff
@@ -17,8 +14,16 @@
...
+        %span.status-wrap.medium-12.small-12{ class: "status-is-" + proposal.status }
           Status:
           %span.status-value
-            = @proposal.capitalized_detailed_status
+            = proposal.capitalized_detailed_status
+          %span.status-icon
+        - if proposal.status == "completed"
+          %span.completed-time
+            on 
+            = proposal.steps.last.updated_at.strftime("%b %d, %Y at %l:%M%P")
```
Comment (pkarman) [inline app/views/proposals/details/_summary.html.haml]: Do you want `.completed_at` instead? That date is more "official" than the `updated_at` date, esp if the step is re-started.

Comment (rememberlenny) [inline app/views/proposals/details/_summary.html.haml]: @pkarman I do! I noticed that the .completed_at wasn't set on the proposal I was working on, so I wasnt sure what was going on. 

Ill update it.

**Comment by rememberlenny** [comment]
@pkarman Updated!

**Comment by nickbristow** [comment]
merged!

**Pull request merged**